vscode/src/client/analyzerClient.ts

```diff
@@ -260,14 +262,8 @@ export class AnalyzerClient {
       // Paths to the Analyzer and jdt.ls
       analyzerLspRpcPath: this.getAnalyzerPath(),
       analyzerLspLspPath: this.assetPaths.jdtlsBin,
-
-      // TODO: With konveyor/kai#509, multiple paths will be accepted
       analyzerLspRulesPath: this.getRulesetsPath(),
-
-      // TODO: Once konveyor/kai#547 is resolved, multiple bundles can be supported
-      // jdt.ls bundles
-      analyzerLspJavaBundlePath: this.assetPaths.jdtlsBundleJars[0],
-
+      analyzerLspJavaBundlePath: this.assetPaths.jdtlsBundleJars,
```

Comment:
Pranav is right -- these params have a trailing `s`.

https://github.com/konveyor/kai/blob/f6103db3b1a1e0c70f2d68e2c4da6e91f5483b3c/kai/rpc_server/server.py#L75-L76

Reply:
Thank you for spelling it out!